MediaGalleryUi/etc/db_schema.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="media_gallery_asset">
+        <column name="source" default="Local"/>
```

Comment:
The Local value should be set when synchronizing or uploading the image from local machine. This will be handled in https://github.com/magento/adobe-stock-integration/pull/1157 Database changes are not needed.